src/chrome/content/rules/store.twentyonepilots.com.xml

```diff
@@ -1,8 +1,10 @@
 <!--
-  Main domain (twentyonepilots.com) unfortunately doesn't supports https
+	Invalid Certificate:
+	- twentyonepilots.com
```

Comment:
And also `www.twentyonepilots.com`

Reply:
Right, forgot that. Thanks for mentioning it.